router/pump.go

```diff
@@ -89,6 +89,15 @@ func (p *LogsPump) Setup() error {
 	return nil
 }
 
+func (p *LogsPump) rename(event *docker.APIEvents) {
+    p.mu.Lock()
+    defer p.mu.Unlock()
+    container, err := p.client.InspectContainer(event.ID)
```

Comment:
We totally could do that. There are a few other spots that use `event.ID` as well. The question is does logspout want to require docker `>= 1.10.0` at this time or should we wait another couple minor API revs?